### DISCUSSION TITLE
[EssentialsX-Discord] Allow slash commands to be used in direct messages

### DISCUSSION THREAD

**Discussion by untuned** [EssentialsX/Essentials#5172] [Ideas and Feature Suggestions]
This would especially be useful for the `/msg` command, since replies are sent to your DMs anyway.

**Comment by Chew** [comment]
This is a Discord limitation. We upsert commands to the server specifically, which means they update in the client instantly. To allow this to happen, commands would need to be global, which take up to an hour to update.

**Reply by untuned** [reply]
In that case, it could be a configuration option? Most bots will only be in one server regardless.
